webpack/assets/javascripts/react_app/common/DeprecationService.js

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-console, max-len */
 
-export const deprecate = (oldMethod, newMethod, version = '1.22') =>
+export const deprecate = (oldMethod, newMethod, version = '1.25') =>
```

Comment:
@tbrisker can you add it to your release checklist? thanks!

Reply:
TBH, i think it would be best to remove the default value here and require specifying release for deprecations

Reply:
I agree I'll update this pr for 1.23 js deprecations

Reply:
ping?